uicomponent-compose/feed/src/main/java/io/github/droidkaigi/feeder/feed/BackLayerContent.kt

```diff
@@ -70,24 +64,14 @@ private fun Input(
                     Color.Transparent
                 }
             )
-            .padding(all = 12.dp)
+            .padding(horizontal = 20.dp, vertical = 12.dp)
```

Comment:
when remove spacers (L83, L90), it seems to go left too much.
so i add additional padding here.